domain/src/main/java/com/coherentsolutions/trainings/java/auto/domain/ProductComparator.java

```diff
@@ -0,0 +1,31 @@
+package com.coherentsolutions.trainings.java.auto.domain;
+
+import app.com.coherentsolutions.trainings.java.auto.store.Product;
+
+import java.util.Comparator;
+
+public class ProductComparator implements Comparator<Product> {
```

Comment:
let's separate this class into PriceComparator, NameComparator, RatingComparator